README.md

```diff
@@ -34,14 +34,37 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+A fully responsive website uses responsive units along with media queries to
```

Comment:
Responsive websites also have hard-coded website widths for each media query BUT they transition smoothly as the viewport width changes. Adaptive websites have a few layouts that do not transition slowly into one another but instead snap from one layout to the next.